tests/testthat/test-heatmap.R

```diff
@@ -0,0 +1,24 @@
+context("Test Heatmap plotting function")
+library("musicatk")
+
+test_that(desc = "Testing Exposures Heatmap Plotting", {
+  result <- readRDS(system.file("testdata", "res_annot.rda", package = "musicatk"))
+  p <- plot_heatmap(result)
+  expect_true(!is.null(p))
+  p <- plot_heatmap(result, proportional = TRUE)
+  expect_true(!is.null(p))
+  p <- plot_heatmap(result, proportional = TRUE,scale = TRUE)
+  expect_true(!is.null(p))
+  p <- plot_heatmap(result, show_column_names = TRUE)
+  expect_true(!is.null(p))
+  p <- plot_heatmap(result, show_row_names = TRUE)
+  expect_true(!is.null(p))
+  p <- plot_heatmap(result, show_column_names = TRUE, show_row_names = TRUE)
+  #p <- plot_heatmap(result, annotation = c("Tumor_Subtypes"))
+  #expect_true(!is.null(p))
+  #p <- plot_heatmap(result, subset_signatures = c("Signature1","Signature2"))
+  #expect_true(!is.null(p))
```

Comment:
What's going on with these lines? If they're not being used they should be removed.